pytorch_trainer/trainer/utils.py

```diff
@@ -155,3 +165,33 @@ def create_work_dir(work_dir):
         raise TypeError('Argument "work_dir" must be a string.')
 
     return work_dir
+
+
+def load_pretained_weight(model, dir_path, weight_name, logger):
+    """Load pretrained weight from path.
+
+    Args:
+        model (nn.module): built model
+        dir_path (str): directory path to weight file
+        model_name (str): name of pretrained weight
+        logger (:obj:`logging`): logger
+    """
+    weight_path = osp.join(dir_path, weight_name)
+    if not os.path.isfile(weight_path):
+        assert False, "Path to pretrained weight does not exist."
+
+    state_dict = torch.load(weight_path,
+                            map_location=lambda storage, loc: storage)
+
+    # [NOTE] Strict mode means the weight should totally match to the model.
+    unexpected, missing = model.load_state_dict(state_dict, strict=True)
```

Comment:
Need exception handler here.
If set strict as True, code below line 187 will never be triggered
When state_dict mismatch error will pop out

Reply:
Resolve by the lastest commit.
* [feat] load weight without strict mode and handle error with assertion.